src/main/java/com/sonyericsson/jenkins/plugins/bfa/ResultFilter.java

```diff
@@ -0,0 +1,20 @@
+package com.sonyericsson.jenkins.plugins.bfa;
+
+import hudson.model.Result;
+
+public class ResultFilter {
+    private static boolean doNotAnalyzeAbortedJobs = false;
+
+    public static void setDoNotAnalyzeAbortedJobs(boolean doNotAnalyzeAbortedJobs) {
+        ResultFilter.doNotAnalyzeAbortedJobs = doNotAnalyzeAbortedJobs;
+    }
+
+    public static boolean needToAnalyze(Result result)  {
+        if (doNotAnalyzeAbortedJobs) {
+            return result != Result.SUCCESS && result != Result.ABORTED;
```

Comment:
Unfortunately, I didn't find a way to do it properly. So, right now we are marking all "timeouted" jobs as failed.